### PR TITLE
Modify S6218: add toString case to Kotlin description

### PR DESCRIPTION
**Pull request by erwan-serandour-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6218/kotlin/rule.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-In data classes, the default behavior of the `equals()` method is to check the equality by field values. This works well for primitive fields or fields, whose type overrides `equals()`, but this behavior doesn't work as expected for array fields.
+In data classes, the default behavior of the ``++equals()++`` method is to check the equality by field values. This works well for primitive fields or fields, whose type overrides ``++equals()++``, but this behavior doesn't work as expected for array fields.
 
 
-By default, array fields are compared by their reference, so overriding `equals()` is highly recommended to ensure a deep equality check. The same applies to the `hashcode()` method.
+By default, array fields are compared by their reference, so overriding``++equals()++`` is highly recommended to ensure a deep equality check. The same applies to the ``++hashcode()++`` and ``++toString()++`` methods.
 
 
-This rule reports an issue if a record class has an array field and is not overriding `equals()` or `hashcode()` methods.
+This rule reports an issue if a record class has an array field and is not overriding ``++equals()++``, ``++hashcode()++`` or ``++toString()++`` methods.
```
Comment (leonardo-pilastri-sonarsource) [inline rules/S6218/kotlin/rule.adoc:9]: Let's rename `record` into `data` classes
```suggestion
This rule reports an issue if a data class has an array field and is not overriding ``++equals()++``, ``++hashcode()++`` or ``++toString()++`` methods.
```

**Review by leonardo-pilastri-sonarsource — CHANGES_REQUESTED** [review]
Just a renaming but it looks good to me!
Also, I noticed that the title of the rule only mentions the equals method, while it is reporting on 3 different methods if they are missing. Do we want to change the rule's title as well? I feel like even if more verbose, it would be more clear to mention all 3 of them.

File: rules/S6218/kotlin/rule.adoc (hunk 1 above)
Comment (erwan-serandour-sonarsource) [reply rules/S6218/kotlin/rule.adoc:9]: nice catch

**Review by leonardo-pilastri-sonarsource — APPROVED** [review]
It's ok for me, I let you decide if we want to change the title of the rule as well, to reflect the complete behavior of it

**Pull request closed without merging**